src/lecturer/components/questionsListView/QuestionsListView.tsx

```diff
@@ -20,34 +27,49 @@ export function QuestionsListView(){
             paddingBottom: "10px",
         },
     })();
-    const onChange = (event:any) => {
+
+    const onChangeImport = (event:any) => {
         var files = event.target.files;
-        var json;
         for (var i = 0, f; f = files[i]; i++) {
```

Comment:
This loop is endless? 
It has assignment instead of conditional expression. ` f = files[i]`